pkg/apis/core/types.go

```diff
@@ -157,6 +157,31 @@ type VolumeSource struct {
 	// CSI (Container Storage Interface) represents ephemeral storage that is handled by certain external CSI drivers (Beta feature).
 	// +optional
 	CSI *CSIVolumeSource
+	// Ephemeral represents a volume that is handled by a normal storage driver (Alpha feature).
+	// The volume will be created and deleted together with the pod ("ephemeral volume").
```

Comment:
> The volume's lifecycle is tied to the pod that defines it - it will be created before the pod starts, and deleted before the pod is removed.

Reply:
Ack.

Reply:
I remembered that the "before the pod is removed" is only true when using foreground deletion, and even then the PV is going to be deleted after the PVC and thus the pod.

My proposal:
- Keep the current behavior and describe it as "The volume's lifecycle is tied to the pod that defines it - it will be created before the pod starts, and deleted when the pod is removed."
- Discuss for beta whether we can make this stricter (tracked here: https://github.com/kubernetes/enhancements/issues/1698#issuecomment-655344680).